src/asciiRegex.js

```diff
@@ -10,7 +10,7 @@ const names = flatten(
   })
 ).join("|");
 
-const edgeCases = ["http", "https"].join("|");
+const edgeCases = "https?\\S*";
```

Comment:
Could you explain me why matching against `\` and `S` literally ?

Reply:
It's not matching literally. There are two backslashes in the string literal because literal backslashes must be escaped there. The string is then converted to a RegExp, where `\S` matches any non-whitespace character.

```javascript
> RegExp("https?\\S*")
/https?\S*/
> RegExp("https?\\S*").exec("https://example.com/foo/bar/baz :)")[0]
'https://example.com/foo/bar/baz'
```

Reply:
Definitely dumb ! 🤣 Forgot about string context, sorry. You are totally right, this is the right approach 👍